osf_tests/test_node.py

```diff
@@ -3525,6 +3525,11 @@ def test_use_as_template_preserves_license(self, project, auth):
         assert new.license.node_license._id == license.node_license._id
         self._verify_log(new)
 
+    def test_can_template_a_registration(self, auth):
+        registration = RegistrationFactory(creator=user)
```

Comment:
If you're trying to use the user fixture you need to pass it in:
`def test_can_template_a_registration(self, user, auth):`

Test is failing. The other test that is failing looks unrelated.